x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/common/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const defaultSearch = '*';
+
+export const QUERY_LANGUAGE_KUERY = 'kuery';
+export const QUERY_LANGUAGE_LUCENE = 'lucene';
+export type QUERY_LANGUAGE = 'kuery' | 'lucene';
```

Comment:
maybe worth to use the ones defined in `src/plugins/data/public/query/lib/get_default_query`

Reply:
It looks like `QueryLanguage` isn't exposed further up in `src/plugins/data/public` and linting doesn't allow importing from further down.